src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const basicDiscount = 20;
+  const extraDiscount = 50;
+
+  if (days < 3) {
+    return rentPrice * days;
+  } else if (days < 7) {
+    return (rentPrice * days) - basicDiscount;
+  } else if (days >= 7) {
+    return (rentPrice * days) - extraDiscount;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) the 3rd point